src/benchmark/config_parser.py

```diff
@@ -336,7 +336,8 @@ def __init__(self, mode, extension, async_request_count, thread_count, stream_co
             self.mode = mode.title()
         else:
             raise ValueError('Mode is required parameter. Mode can only take values: '
-                             'Sync, Async, OVBenchmark_Python_Latency, OVBenchmark_Python_Throughput.')
+                             'Sync, Async, OVBenchmark_Python_Latency, OVBenchmark_Python_Throughput, '
```

Comment:
Now ValueError is not necessary here,  so remove it

Reply:
Removed